Snakefile

```diff
@@ -16,40 +19,40 @@ rule files:
 files = rules.files.params
 
 rule download:
-    message: "Downloading sequences from fauna"
+    message: "Downloading sequences and metadata from data.nextstrain.org"
     output:
-        sequences = "data/ebola.fasta"
+        sequences = "data/sequences.fasta",
+        metadata = "data/metadata.tsv",
     params:
-        fasta_fields = "strain virus accession collection_date region country division location source locus authors url title journal puburl"
+        sequences_url = "https://data.nextstrain.org/files/workflows/ebola/test/sequences_all.fasta.zst",
+        metadata_url = "https://data.nextstrain.org/files/workflows/ebola/test/metadata_all.tsv.zst",
     shell:
         """
-        python3 ../fauna/vdb/download.py \
-            --database vdb \
-            --virus ebola \
-            --fasta_fields {params.fasta_fields} \
-            --resolve_method choose_genbank \
-            --path $(dirname {output.sequences}) \
-            --fstem $(basename {output.sequences} .fasta)
+        curl -fsSL {params.sequences_url:q} --output {output.sequences}
+        curl -fsSL {params.metadata_url:q} --output {output.metadata}
         """
 
-rule parse:
-    message: "Parsing fasta into sequences and metadata"
+rule wrangle_metadata:
     input:
-        sequences = files.input_fasta
+        metadata=rules.download.output.metadata,
```

Comment:
Although using rule variables will not prevent a user from bypassing or inserting their own rules, I can see how setting "explicit paths" may make it easier for a user to find the relevant input filesnames.

Done with commit: 92c5f56af3fd8461bfc32e84f86008f787124221

Reply:
> Also, I just confirmed that this workflow works as expected from the compressed inputs, as long as the environment has pandas >=1.4.0 installed ([that's the version where read_csv added zstd support as a default](https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html) instead of an optional dependency).

Thanks for checking!